src/commands/push-to-app-collection.yaml

```
@@ -106,12 +80,12 @@ steps:
         - run:
             name: Add user secret <<parameters.user_secret_path>> into <<parameters.app_collection_repo>>
             command: |
-              mv -n <<parameters.user_secret_path>> .app-collection/helm/<<parameters.app_collection_repo>>-chart/templates/$(cat .app_user_secret_file_name)
-              cd .app-collection && git add helm/<<parameters.app_collection_repo>>-chart/templates/$(cat ../.app_user_secret_file_name)
+              mv -n <<parameters.user_secret_path>> .app-collection/helm/<<parameters.app_collection_repo>>-chart/templates/<<parameters.app_name>>-unique.user_secret.yaml
+              git -C .app-collection add helm/<<parameters.app_collection_repo>>-chart/templates/<<parameters.app_name>>-unique.user_secret.yaml
   - run:
       name: Commit changes
       command: |
-        cd .app-collection && git commit -m "$(cat ../.git_commit_message)"
+        git -C .app-collection commit -m "update <<parameters.app_name>>-unique to $(cat ../.version)"
```

Heh since -C is used then the `../` became a wrong dir :D 
```suggestion
        git -C .app-collection commit -m "update <<parameters.app_name>>-unique to $(cat .version)"
```